sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/kafka/AzureEventHubsKafkaBinderOAuth2AutoConfigurationTest.java

```diff
@@ -69,6 +71,37 @@ void shouldConfigureWhenBinderNameSpecified() {
                     assertThat(context).hasSingleBean(BindingServiceProperties.class);
 
                     testBinderSources(context.getBean(BindingServiceProperties.class), "kafka", AZURE_KAFKA_SPRING_CLOUD_STREAM_CONFIGURATION_CLASS);
+                    assertEquals("value", context.getBean(BindingServiceProperties.class).getBinders().get("kafka").getEnvironment().get("key"));
+                });
+    }
+
+    @Test
```

Comment:
How about adding a test class for BindingServicePropertiesBeanPostProcessor? These internal logic should be as cases in that test class as well.